app.py

```
@@ -152,4 +152,4 @@ def get_balance(account_id):
         return jsonify({"error": str(e)}), 500
 
 if __name__ == '__main__':
-    app.run(host='0.0.0.0', port=5000, debug=True)
\ No newline at end of file
+    app.run(host='0.0.0.0', port=5000, debug=True)
```

Ruff (0.12.2)

Possible binding to all interfaces

(S104)